pootle/apps/pootle_store/syncer.py

```diff
@@ -235,10 +243,10 @@ def create_store_file(self, last_revision, user):
         store = self.convert()
         if not os.path.exists(os.path.dirname(self.store_file_path)):
             os.makedirs(os.path.dirname(self.store_file_path))
+        self.store.file = self.relative_file_path
```

Comment:
Previously this set an absolute file path, whereas now it is a relative one. Correct me if I'm wrong, but is this now possible because the `FieldFile` descriptor in `store.file` is using `PootleFileSystemStorage`, which sets its base location to `POOTLE_TRANSLATION_DIRECTORY`? If so, please let's add a comment in the commit message to make it clear.